euleros/maintainer.md

```diff
@@ -0,0 +1 @@
+Li Ruilin liruilin4@huawei.com (@Kazero0)
```

Comment:
This will be listed directly on https://hub.docker.com/_/euleros/ under "Maintained by:" -- should this be your personal email, or simply something like "Huawei Engineering" or some similar team / group / project name?

(See other image folders in this repo for examples of the typical format of this file.)

Reply:
I'll change it to the `The EulerOS Project`.